src/inlineeditor.js

```diff
@@ -73,7 +73,9 @@ export default class InlineEditor extends Editor {
 			secureSourceElement( this );
 		}
 
-		const view = new InlineEditorUIView( this.locale, this.editing.view, this.sourceElement );
+		const shouldToolbarGroupWhenFull = !this.config.get( 'toolbar.shouldNotGroupWhenFull' );
+
+		const view = new InlineEditorUIView( this.locale, this.editing.view, this.sourceElement, { shouldToolbarGroupWhenFull } );
```

Comment:
[Let's break it into 2 lines to improve readability.](https://github.com/ckeditor/ckeditor5-editor-classic/blob/master/src/classiceditor.js#L74-L76)

Reply:
:ok: